src/components/UpdateArticleVotes.jsx

```diff
@@ -7,20 +7,19 @@ function UpdateArticleVotes (){
     const article = useParams()
     const articleChoice = article.article_id
     let votes = 0
-    let [newVotes, setNewVotes] = useState(votes)
-    useEffect(()=>{
+    const [newVotes, setNewVotes] = useState(votes)
+
+    const handleSubmit = (e)=>{
         axios.patch(`https://northcoders-news-api-bjpy.onrender.com/api/articles/${articleChoice}`,{inc_votes: newVotes})
     .then((response)=>{
```

Comment:
We're missing something here. Take a look at the notes for optimistic rendering.